ui/app/AppLayouts/Chat/views/communities/CommunityNewPermissionView.qml

```diff
@@ -515,6 +517,18 @@ StatusScrollView {
                 onToggled: d.dirtyValues.isPrivateDirty = (root.isPrivate !== checked)
             }
         }
+
+        PermissionQualificationPanel {
+            Layout.fillWidth: true
+            Layout.topMargin: 24
+
+            visible: d.dirtyValues.holdingsModel.count > 0
+
+            qualifyingAddresses: 200234
```

Comment:
I was thinking about that but concluded that we don't know too much about how it will be exposed from the store. So maybe it's better to wait a bit with that.